### PR TITLE
Value Formula and Cost

### PR DESCRIPTION
**Pull request by SeverTopan**
Implementation of our recent discussion. See https://severtopan.github.io/Vulcan-Academy.github.io/Lexicon.html for formatted version.

### CODE REVIEW

File: Lexicon.md [hunk 1]
```diff
@@ -14,10 +14,26 @@
...
+* __World State__: Given that the __Information__ in an __Environment__ may change with respect to some parameter (for example, time), __World State__ defines a static snapshot of the __Information__ within an __Environment__. In other words, a __World State__ is an __Environment__ where all __Information__ is held constant. Denoted $$ w \in W $$.
 * __World Model__: An Agent's internal __Model__ of its __World__. 
 * __Action__: An __Event__ in which __Information__ exits an Agent.
 * __Action Space__: The set of all __Actions__ that a particular Agent may perform. All __Actions__ performed by an Agent are sampled from its __Action Space__.
-* __Value__: An Agent's expression of preference between two __World States__. __Value__ may favour either __State__, or may reflect indifference. __Value__ is not intrinsic, and cannot be specified in the absence of an __Actor__.
+* __Value__: An Agent's expression of preference between two __World States__. __Value__ may favour either __State__, or may reflect indifference. __Value__ is not intrinsic, and cannot be specified in the absence of an Agent. Given an Agent's __Value__ system $$ s $$, __Value__ is formulated as $$
```
Comment (macterra) [inline Lexicon.md]: Maybe rename this to Preference?

File: Lexicon.md [hunk 2]
```diff
@@ -14,10 +14,26 @@
...
+\begin{array}{cl}
+0 & w_1 \ {\rm and} \ w_2 \ {\rm of \ equal \ value}\\
+1 & w_1 \ {\rm of \ greater \ value}\\
+-1 & w_2 \ {\rm of \ greater \ value}
+\end{array}
+\right.
+\end{equation}
+$$
 * __Valuation__: The process of assigning a __Value__.
+* __Cost__: A measurement of the aggregate __Value__ of a particular __World State__. Given Agent __Value__ system $$ s $$ and __World__ $$ W $$ with $$ N_W$$ __States__, __Cost__ is defined as $$
```
Comment (macterra) [inline Lexicon.md]: Maybe rename Cost to Value (assuming Value is renamed to Preference)?

**Review by macterra — COMMENTED** [review]
I tried a simple example to see how this might work in practice
https://docs.google.com/spreadsheets/d/1aPffjpL0EZwjsrEGJ2wE3WIOaFq7CPD2Pog_pJLYKeE/edit?usp=sharing

**Comment by macterra** [comment]
image

The world states are (the agent possessing) $1, $2, $3, $4, $5, $6, 1 pie, 2 pies, 3 pies, 4 pies

**Comment by macterra** [comment]
In this example the agent values the pie at $3, the 2nd pie at $2, the 3rd pie at $1, and zero for the 4th pie.

File: Lexicon.md (hunk 1 above)
Comment (SeverTopan) [reply Lexicon.md]: I agree with the rename, it's more intuitive. The only reason I can think of to keep the current naming is that the current definition of __Value__ is the same as Mises' definition of Value.

**Comment by SeverTopan** [comment]
From the value array it looks like p1 == $3, p2 == $5, p3 == $6, and p4 is worth >$6.

Also if my interpretation from above is correct, cell J7 should be 0 so that 3p and $6 have the same Cost (current nomenclature).

File: Lexicon.md (hunk 1 above)
Comment (macterra) [reply Lexicon.md]: citation to Mises?

Comment (SeverTopan) [reply Lexicon.md]: https://wiki.mises.org/wiki/Subjective_theory_of_value

Comment (SeverTopan) [reply Lexicon.md]: Note that Mises claims that Value is ordinal, not cardinal (that is why he claims that it is a comparitive function).

**Comment by macterra** [comment]
image
good catch, fixed. 

p4 is worth the same as p3 because the marginal value of additional pies has dropped to 0

File: Lexicon.md (hunk 1 above)
Comment (macterra) [reply Lexicon.md]: Right but maybe ordinals can be converted to cardinals by introducing a common denominator (i.e. medium of exchange, namely money). That is why my example includes dollars.

File: Lexicon.md (hunk 2 above)
Comment (macterra) [reply Lexicon.md]: OTOH if you want to keep Value then this should be renamed to something other than Cost which has distinctly different (but related) connotations associated with investment and/or sacrifice.

**Review by macterra — APPROVED** [review]

**Review by cecilyth — APPROVED** [review]

**Pull request merged**